src/node_crypto.cc

```diff
@@ -3515,11 +3549,14 @@ SignBase::Error Sign::SignInit(const char* sign_type) {
 
 void Sign::SignInit(const FunctionCallbackInfo<Value>& args) {
   Sign* sign = Unwrap<Sign>(args.Holder());
+  Environment* env = sign->env();
 
-  if (args.Length() == 0 || !args[0]->IsString()) {
-    return sign->env()->ThrowError("Must give signtype string as argument");
+  if (args.Length() == 0) {
+    return env->ThrowError("Signtype argument is mandatory");
```

Comment:
I realize you copied this from the existing error message but you write 'Sign type' a few lines down.